### PR TITLE
Added test case to validate message with apostrophe and arguments

### PR DESCRIPTION
**Pull request by rrameshbtech**
Based on my understand of below PR, I have added one test to expect a message with apostrophe & arguments. This message has single apostrophe. This test case fails.

https://github.com/mekomsolutions/openmrs-module-initializer/pull/69

@ibacher , @mks-d is this test case expected to fail?

### CODE REVIEW

File: api/src/test/resources/testAppDataDir/configuration/messageproperties/metadata_fr.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
 metadata.healthcenter=Clinique
 metadata.healthcenter.description=Ceci est la description d'une clinique.
 metadata.healthcenter.description.named=Ceci est la description de la clinique {0}.
+metadata.healthcenter.details=Voici l'exemple d'apostrophe {0}.
```
Comment (mks-d) [inline api/src/test/resources/testAppDataDir/configuration/messageproperties/metadata_fr.properties:4]: @rrameshbtech so this is showing that single quote + argument works, thereby contradicting @ibacher's remark [here](https://github.com/mekomsolutions/openmrs-module-initializer/pull/69#issue-426185375), correct?

Comment (rrameshbtech) [reply api/src/test/resources/testAppDataDir/configuration/messageproperties/metadata_fr.properties:4]: Sorry. I missed. This is failing test case.

Comment (mks-d) [reply api/src/test/resources/testAppDataDir/configuration/messageproperties/metadata_fr.properties:4]: @rrameshbtech if the test fails then that's in line with @ibacher's conclusion.

Does this make the test pass?
```java
metadata.healthcenter.details=Voici l''exemple d'apostrophe {0}.
```
So with the double `''`.

Comment (ibacher) [reply api/src/test/resources/testAppDataDir/configuration/messageproperties/metadata_fr.properties:4]: @rrameshbtech As I remarked in the link @mks-d pointed out, this is odd, but (it seems) expected behaviour. See [the Javadoc for `AbstractMessageSource#resolveCodeWithoutArguments(String, Locale)`](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/support/AbstractMessageSource.html#resolveCodeWithoutArguments-java.lang.String-java.util.Locale-).

**Pull request closed without merging**